### PR TITLE
Add postgres support

### PR DESCRIPTION
**Pull request by patelh**
Added postgres support to core.

Added new postgres executor, @pranavbhole need to push this to bintray, can you please do the needful.

One breaking change in service config for registry config, need to add new config to registry, e.g.:
```
      "defaultFactEngine": "oracle",
      "druidMultiEngineQueryList": ["oracle"]
```

I confirm that this contribution is made under the terms of the license found in the root directory of this repository's source tree and that I have the authority necessary to make this contribution on behalf of its copyright owner.

### CODE REVIEW

**Comment by patelh** [comment]
@pranavbhole @pavanab4u @panditsurabhi please review.  Will need to add new postgres module to bintray so it publishes.

File: core/src/main/scala/com/yahoo/maha/core/query/oracle/OracleOuterGroupByQueryGenerator.scala [hunk 1]
```diff
@@ -488,22 +488,28 @@ abstract class OuterGroupByQueryGenerator(partitionColumnRenderer:PartitionColum
...
-                    factAlias -> name
-                }
-                require(aliasToColNameMap.contains(alias), s"Can not find the alias $alias in aliasToColNameMap")
-                val colName = aliasToColNameMap(alias)
-                queryBuilderContext.getFactColByAlias(colName)
-              }
+            case FactColumnInfo(alias) if queryBuilderContext.containsFactAliasToColumnMap(alias) =>
+              val column = queryBuilderContext.getFactColByAlias(alias)
               renderParentOuterDerivedFactCols(alias, column)
+              /*TODO: I am unable to generate a test case for the else condition below with any combinations of
```
Comment (patelh) [inline core/src/main/scala/com/yahoo/maha/core/query/oracle/OracleOuterGroupByQueryGenerator.scala]: @pranavbhole I couldn't create a test case with any combination of custom rollup and derived column to get the test to get into this if statement.  If you have an example, please add to this branch, I gave you access to my repo so you can push to this branch (github.com/patelh/maha).

**Comment by patelh** [comment]
@pranavbhole @pavanab4u @panditsurabhi please review.   Will need to add new postgres module to bintray so it publishes.

**Comment by patelh** [comment]
@pranavbhole @pavanab4u @panditsurabhi please review, we're at 2 weeks now. Will need to add new postgres module to bintray so it publishes.

**Comment by pranavbhole** [comment]
working on getting package in bintray

**Comment by pranavbhole** [comment]
package entitlement has been created for maha-postgres-executor. We can merge this PR once build passes, restarted the build.

**Comment by patelh** [comment]
@pavanab4u @panditsurabhi @pranavbhole can we merge if we are ready?

**Pull request merged**